int32 ts
float64 lat
float64 lon
float32 alti
float32 height
float32 uncertain